### DISCUSSION TITLE
Shimmer

### DISCUSSION THREAD

**Discussion by Shamuan** [iotaledger/firefly#2955] [Help & Support]
Lost 9m shimmer after wallet update

**Comment by MiKeF05** [comment]
What do you exactly mean with you lost 9b shimmer?

**Reply by Shamuan** [reply]
Previously,  displayed 9m shimmers more, after the update -9m from the original amount

**Reply by MiKeF05** [reply]
Have you staked with several wallets in the last staking period?

**Reply by Shamuan** [reply]
I staked Iota 90 days with my wallet only and earlier I was supposed to get 50 million, but now 41, maybe some error

**Reply by MiKeF05** [reply]
Did you have several wallets in your profile?

**Reply by Shamuan** [reply]
No, one wallet

**Reply by MiKeF05** [reply]
So you had 1 profile and 1 wallet and you only staked with 1 address?

**Reply by Shamuan** [reply]
Yes

**Reply by MiKeF05** [reply]
Can you try to run the balance finder in the settings a few times?

**Reply by Shamuan** [reply]
i tried but nothing

**Reply by MiKeF05** [reply]
It would be good if you can join the official IOTA discord server and go to the #help channel. But be aware to never share any info whenever someone contacts you.

**Reply by Shamuan** [reply]
ok, thx

**Reply by Arkiteker** [reply]
I have the exact same issue, the ASMB and SMR balances are WAY off. WTF is going on?

**Reply by MiKeF05** [reply]
If you have staked with more than 1 wallet then you need to switch to the other wallet to see the rewards for the other wallet.

**Comment by Arkiteker** [comment]
How do I move them into a single wallet like I did my mIOTA?

**Reply by MiKeF05** [reply]
The Shimmer and Assembly networks will go live. Then you are able to send them.

**Comment by Arkiteker** [comment]
Okay. Thank you very much for your help!
